text/0002-split-coinstake-transaction/0002-split-coinstake-transaction.md

```diff
@@ -37,21 +35,23 @@ Only the coinbase `vin[0]` has been used to carry meta-data (e.g. signaling P2SH
 Splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction, has the side effect of creating small value outputs.
 Because those outputs are very unlikely to mint new blocks, node implementations are adviced to join them with a future *coin-age consumption* transaction.
 
-The minter is awarded a budget of 1 KB for the *coin-age consumption* transaction in order to adjust their UTXO table as they see fit.
-If the *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
+The minter is currently awarded a budget of 1 KB for the coinstake transaction, and will continue to have this power.  If a *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
 
 The diagram below illustrates the block layout with a splitted coinstake transaction.
 
 ![splitted block layout](split.png)
 
 ## Advantages
 
-* Paving the road for other improvements like multi-signature minting.
-* Limiting the power of minters to spam the blockchain.
+* Required for multi-signature minting, as described in [RFC-0003](../0003-multisig-minting/0003-multisig-minting.md)
+* While there is no known advantage for aligning the transaction structure with Bitcoin's, it should be taken as a step toward simplification, which can foster innovation.
 
 ## Drawbacks
 
-* Hard fork
+* Dependency on [RFC-0004](../0004-remove-transaction-timestamp/0004-remove-transaction-timestamp.md).
+* Hard fork of protocol rules.
+* Previous transaction formats will need to be supported beyond the hard fork, until all transaction creation software switches over.
+* Previous block formats will need to be supported by full nodes as long as the chain record persists.
```

Comment:
As is the case with any protocol fork.

Reply:
Any protocol fork that changes the transaction or block format, yes.  Actually, now that I think about it, this RFC doesn't change the transaction format, so that bullet point can be removed.  Supporting both block formats on full nodes, however, is definitely something that will need to happen.

An example of a hard fork that doesn't change the block format would be switching from coindays to mint difficulty for chain weight.

Reply:
Actually this is the same, the block format would not change here either. There still is a coinbase and a coinstake transaction serialized in exactly the same way, so the implications are equal to any other protocol change like the one you mention.

Reply:
Ah, I see, so you can add outputs to the coinbase without breaking protocol, but you can't add inputs.  So moving the block reward to the coinbase doesn't change the format, it only changes the convention and the calculation of output amounts.

Reply:
This change does break the protocol, just like your chain weight advantage.
What does not change is the serialization protocol. (The way blocks and transactions are serialized)